BLUEPRINT/reactor.py

```diff
@@ -292,13 +292,13 @@ def run_systems_code(self):
         """
         Runs, reads, or mocks the systems code according to the build config dictionary.
         """
-        PROCESS_output: ParameterFrame = run_systems_code(
+        PROCESS_solver = run_systems_code(
```

Comment:
:smile: I've been doing the same 2 line interface change elsewhere, definitely agree